sql_script/MS3_insertions.sql

```diff
@@ -1,41 +1,42 @@
-/*INSERTIONS___
-#Add new inventory types to the inventory_type table*/
-INSERT
-INTO inventory_type(type_name, height, width, length)
-VALUES (new_type_name, new_height, new_width, new_length);#impact on dimensions table?
-
-/*Add a new recipient to the recipient table*/
-INSERT
-INTO recipient(id, street_address, postal_code)
+/*Add new inventory types to the inventory_type table 
+-> NOTE: only allows insertion when dimensions already exist in dimensions table*/
+INSERT INTO inventory_type(type_name, height, width, length)
+VALUES (new_type_name, new_height,new_width,new_length);
+
+/*Add a new recipient to the recipient table 
+-> NOTE: only allows insertion if postal code already exists in region*/
+INSERT INTO recipient(id, street_address, postal_code)
 VALUES (new_id, new_street_address, new_postal_code);
 
-/*Add a new producer to the producer table*/
+/*Add a new producer to the producer table
+-> NOTE: only works when new_postal_code already exists in region table*/
 INSERT
 INTO producer (id, street_address, postal_code)
 VALUES (new_id, new_street_address, new_postal_code);
 
-/*!!!!!!!!!!!!!!!Come up with values for a new container type, calculate its volume, insert that value into the dimensions table, then insert the container into container_type (has a FK to dimensions).
-*/
-INSERT
-INTO storage_container (id, warehouse_id, occupied_volume, height, width, length)
-VALUES (new_id, new_warehouse_id, 0, new_height, new_width, new_length);
+/*Come up with values for a new container type, calculate its volume, insert that value into the dimensions table, then insert the container into container_type (has a FK to dimensions).
+-> NOTE: only works if height, width, length tuple already exists in dimension table*/
 
-	/*check if it exists in dimensions?*/
 INSERT
-INTO dimensions(height, width, length, volume)
-VALUES (new_height, new_width, new_length, (new_height*new_width*new_length));
+INTO storage_container (id, warehouse_id, occupied_volume, height, width, length)
+VALUES (new_id, new_warehouse_id, new_occupied_volume, new_height, new_width, new_length);
 
-/* !!!!!!!!!!!!!!!!!!! Insert inventory, assign it to a container, update the container’s occupied volume, then update the warehouse’s total volume (??)
-*/
+/*Insert inventory, assign it to a container, update the container’s occupied volume, then update the warehouse’s total volume (??)
+-> NOTE: only works if new_type_name already exists in type_name table*/
 INSERT
 INTO inventory (serial_num, container_id, type_name, weight, manufacture_date)
```

Comment:
Recommend putting these `INSERT INTO`s on the same line.

regex: `INSERT\nINTO` replace with: `INSERT INTO`